src/QueryDescription/index.js

```diff
@@ -256,15 +270,19 @@ export function where(left: ColumnName, valueOrComparison: Value | Comparison):
 }
 
 export function unsafeSqlExpr(sql: string): SqlExpr {
-  invariant(typeof sql === 'string', 'Value passed to Q.unsafeSqlExpr is not a string')
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Why only not in production? Shouldn't this trigger a runtime exception?

Reply:
I figured that is unlikely to get triggered outside of development - it would because you misused the API by accident. If someone passes unsanitized SQLs to Q (not a string literal), then a) it will just throw on query, b) you likely have bigger issues anyway. Runtime checks affect code size and runtime performance, so generally they're all wrapped in production check in WatermelonDB so they can be stripped entirely for production...